disk/test/com/treode/disk/edit/DiskIOSpec.scala

```diff
@@ -18,44 +18,56 @@ package com.treode.disk.edit
 
 import com.treode.async.io.stubs.StubFile
 import com.treode.async.stubs.StubScheduler
-import com.treode.async.stubs.implicits._
 import com.treode.disk.DiskTestConfig
 import org.scalatest.FlatSpec
+import com.treode.pickle.Picklers._
+import com.treode.disk.PageDescriptor
+import com.treode.disk.Position
+import com.treode.async.{Async, Callback, Fiber, Scheduler }, Async.async
+import com.treode.disk.PickledPage
+import com.treode.async.stubs.implicits._
+import com.treode.pickle.Picklers
+
 
 class DiskIOSpec extends FlatSpec {
 
   implicit val config = DiskTestConfig()
-
-  "The PageReader" should "be able to read the string the PageWriter wrote" in {
+   "The PageReader" should "be able to read the tuple the PageWriter wrote" in {
     implicit val scheduler = StubScheduler.random()
-    val a = "this is a string"
+
+    val stringPickler = 
```

Comment:
Why test with a `(String, String)` pickler? I think it's sufficient to just test with Picklers.string or Picklers.int.